ocrd_lib.sh

```diff
@@ -51,7 +51,6 @@ init() {
   # try to be unique here (to avoid clashes)
   REMOTEDIR="KitodoJob_${PID}_$(basename $PROCESS_DIR)"
 
-  WORKFLOW=$(command -v "$WORKFLOW" || realpath "$WORKFLOW")
```

Comment:
> Maybe it's related to [this change](https://github.com/slub/ocrd_kitodo/pull/98/files#diff-8764598e9de17d07d9dbf78eba8da3202a4f1880bf6cc1f4f5fce2e3d3617af6) on the other side?

Assuming it is: I can see how one would be tempted to think that workflows must already have absolute names on the caller's side, because whatever the CWD of the caller is might not translate into the SSH session on the Manager. But there may be scenarios where a relative path actually does work and can be expected to work. Removing the above line would break that, while keeping it does not break absolute names in any way.

So I still suggest:
```suggestion
  WORKFLOW=$(command -v "$WORKFLOW" || realpath "$WORKFLOW")
```